oracle/internal/helpers.go

```diff
@@ -110,7 +116,7 @@ func DeserializeDkgR2(r2Packages map[uint16][]byte /*map[FROM]data*/, vsetMask *
 		}
```

Comment:
Why don't we consider this an error if we have destination `validatorIdx` excluded from the vset mask?